explorer/ast/ast_test_matchers_test.cpp

```diff
@@ -23,7 +23,7 @@ using ::testing::Not;
 
 static constexpr SourceLocation DummyLoc("dummy", 0, FileKind::Main);
 
-TEST(BlockContentsAreTest, BasicUsage) {
+TEST(ASTTestMatchers, BlockContentsAreTest) {
```

Comment:
Might make sense to split this off as a trivial improvement to the test you can easily land first?